Extensions/StyleCop/StyleCopTask/task.json

```diff
@@ -3,11 +3,11 @@
   "name": "StyleCop",
   "friendlyName": "StyleCop Runner by Black Marble",
   "description": "Black Marble's StyleCop test runner that use settings files in the project folders. (Using StyleCop 4.7.59.0)",
- "helpMarkDown": "Version: #{Build.BuildNumber}#. [More Information](https://github.com/rfennell/vNextBuild/wiki/StyleCop-Runner-Task/)",
+  "helpMarkDown": "Version: #{Build.BuildNumber}#. [More Information](https://github.com/rfennell/vNextBuild/wiki/StyleCop-Runner-Task/)",
```

Comment:
I can do but looking at the change it actually fixes missing indentation so might be preferential (this was IDE added, I hadn't noticed).